packages/create-react-native-library/src/template.ts

```diff
@@ -137,10 +142,12 @@ export function generateTemplateConfiguration({
               .replace(/[^a-z0-9](\w)/g, (_, $1) => $1.toUpperCase())
               .slice(1)}`,
       package: pack,
+      package_array: JSON.stringify(pack.split('.')),
       package_dir: pack.replace(/\./g, '/'),
       package_cpp: pack.replace(/\./g, '_'),
       identifier: slug.replace(/[^a-z0-9]+/g, '-').replace(/^-/, ''),
       native: languages !== 'js',
+      nitro: type.startsWith('nitro'),
```

Comment:
Keep in mind that string matching means refactors won't be caught by TS. Prefer explicitly checking the string with equality operator.